Class/B-Passing-Objects-to-Functions/exercise2.js

```diff
@@ -25,8 +25,18 @@ let student3 = {
 // set hasPassed to true and return the String 
 // "Student " + name + " has passed"
 // Otherwise return the String "Student " + name + " has failed"
-
+function getGrade(student){
+    if(student.attendance >= 90 && student.examScore > 60){
+        hasPassed = true;

```

Comment:
Hey, Megumi, 
what is `hasPassed` here?
do `student2` and `student3`pass the test?

If `student1` looks like passed the test, after you ran `getGrade` function for this object, 
does `student1` has their grade updated?
```javascript
console.log(getGrade(student1));
console.log(student1.hasPassed);
```